docs/syntax.md

```diff
@@ -134,7 +134,36 @@ export let name;
 | Special Characters           | `&nbsp;`                                 | `{'\xa0'}` or `{String.fromCharCode(160)}`         |
 | Attributes                   | `dash-case`                              | `camelCase`                                        |
 
-### TODO: Styling
+### URL resolution
+
+It’s important to note that Astro **won’t** transform your URLs for you. For example, say you have the following structure:
```

Comment:
can we be more specific? "URLs and references inside of HTML element attributes"? Snowpack does resolve import URLs, for example.

Reply:
Good point; this is ambiguous